packages/babel-plugin/__tests__/stylex-transform-alias-config-test.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ */
+
+'use strict';
+
+import path from 'path';
+import fs from 'fs';
+import os from 'os';
+import StateManager from '../src/utils/state-manager';
+
+describe('StyleX Alias Configuration', () => {
```

Comment:
Please write tests against the public API. If we refactor the internals, tests like this would likely need to be rewritten. And this test doesn't actually validate that real code is correctly processed.

Reply:
Will do. Will need to simulate a filesystem or similar to make this happen, but should be doable.